backend/src/main/java/com/benine/backend/http/CameraFocusHandler.java

```diff
@@ -0,0 +1,50 @@
+package com.benine.backend.http;
+
+import com.benine.backend.LogEvent;
+import com.benine.backend.camera.CameraConnectionException;
+import com.benine.backend.camera.FocussingCamera;
+import org.eclipse.jetty.server.Request;
+
+import java.io.IOException;
```

Comment:
I see jetty and javafx imports for http things should these two be mixed together?